queue_services/payment-reconciliations/src/reconciliations/worker.py

```diff
@@ -134,8 +140,8 @@ async def _update_payment_details(msg: Dict[str, any]):
                 # NSF Condition. Publish to account events for NSF.
                 _process_failed_payments(payment, row)
                 # Send mailer and account events to update status and send email notification
-                await _publish_account_events('PaymentFailed', payment_account, row)
-                await _publish_mailer_events('PaymentFailed', payment_account, row)
+                await _publish_account_events('lockAccount', payment_account, row)
+                await _publish_mailer_events('lockAccount', payment_account, row)
```

Comment:
we can think of moving this to account -events as well.. just to be in sync ..incase if mail goes before auth-db is updated

Reply:
Done